assets/src/edit-story/elements/text/index.js

```diff
@@ -34,8 +34,27 @@ export const defaultAttributes = {
   ...SHARED_DEFAULT_ATTRIBUTES,
   backgroundTextMode: BACKGROUND_TEXT_MODE.NONE,
   bold: false,
-  fontFamily: 'Roboto',
-  fontFallback: ['Helvetica Neue', 'Helvetica', 'sans-serif'],
+  font: {
+    family: 'Roboto',
+    weights: [100, 300, 400, 500, 700, 900],
+    styles: ['italic', 'regular'],
+    variants: [
```

Comment:
This is a little concerning, although probably ok as a first pass. I think longterm we should just have a sparse `font{family}` for this case and then `useInsertElement` should query and expand `font{}` struct with the rest of font arguments. But, until we have a need to support more fonts this way, we can get away with constants. But let's move them into a single independent file somewhere closer to `app/font`.

Reply:
Moved to a constant here: 96dc787717174b744c181a98063b7f91bff6f115